### PR TITLE
INTEGRA-4149 Update AC Open Source library with latest AC version from Internal Repo

### PR DESCRIPTION
**Pull request by danbuga**

### CODE REVIEW

File: java/anaplan-connect-library/api-core/src/test/java/com/anaplan/client/listwriter/ListItemTest.java [hunk 1]
```diff
@@ -95,4 +95,4 @@ void testInitListImpl() {
     assertEquals("b", result.get(0));
     assertEquals(1, result.size());
   }
-}
+}
```
Comment (skharab-anaplan) [inline java/anaplan-connect-library/api-core/src/test/java/com/anaplan/client/listwriter/ListItemTest.java]: Nit: EOF new line

File: java/anaplan-connect-library/api-core/src/main/java/com/anaplan/client/Utils.java [hunk 2]
```diff
@@ -402,17 +369,6 @@ public static Key loadKeystore(FileInputStream fileInputStream, String keyStoreN
...
-   * @param encodedKey
-   * @param keyStoreName
-   * @param password
-   * @return {@link KeyStore}
-   * @throws KeyStoreException
-   * @throws IOException
-   * @throws NoSuchAlgorithmException
-   * @throws CertificateException
-   */
   public static KeyStore saveEntryInKeyStore(byte[] encodedKey, String keyStoreName, char[] password)
```
Comment (skharab-anaplan) [inline java/anaplan-connect-library/api-core/src/main/java/com/anaplan/client/Utils.java]: Please don't remove comments and java docs.

Comment (danbuga) [reply java/anaplan-connect-library/api-core/src/main/java/com/anaplan/client/Utils.java]: fix

File: java/anaplan-connect-library/api-core/src/test/java/com/anaplan/client/listwriter/ListItemTest.java (hunk 1 above)
Comment (danbuga) [reply java/anaplan-connect-library/api-core/src/test/java/com/anaplan/client/listwriter/ListItemTest.java]: fixed

**Review by ajunnola-anaplan — APPROVED** [review]

**Pull request merged**